appveyor.yml

```
@@ -86,3 +86,4 @@ deploy:
     description: "Release of $(SOLUTION_NAME) - v$(appveyor_build_version)"
     auth_token: $(GITHUB_TOKEN)
     force_update: true
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)